src/templates/blog-post.tsx

```diff
@@ -16,7 +16,7 @@ import { LocalesQuery } from '../pages';
 import FollowPanel from '../components/social-panel/follow-panel';
 import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { Astronaut } from '../components/icons/illustrations/astronaut';
```

Comment:
Also an unused import.

Reply:
I can tell my IDE to clean up. I think I should do this on save of even better we have some commit hook for things like that too. Thanks for the catch here too.